### ISSUE TITLE
Finish warning to user when deleting index points

### ISSUE THREAD

**Issue by codymccoy** [HydrologicEngineeringCenter/HEC-FDA#488]
We want to warn the user that any stage damages that use these index points will be deleted.
![image](https://user-images.githubusercontent.com/47757848/184361281-d2a3a347-d891-4c6c-acf8-5571164f4599.png)

**Comment by codymccoy** [comment]
Completed in stage_damage_warning_message branch. I added this warning message if removing index points that are being used by a computed stage damage element that uses those index points.
![image](https://user-images.githubusercontent.com/47757848/191109279-2b6b228d-a4d4-4745-9b18-41f46d5b17f7.png)

If the user clicks "Yes", the index points will be deleted and the stage damage will be deleted.

**Comment by codymccoy** [comment]
Completed in stage_damage_warning_message branch. I added this warning message if removing index points that are being used by a computed stage damage element that uses those index points.
![image](https://user-images.githubusercontent.com/47757848/191109573-d0eb6769-8cb5-452b-9244-0df8d2a7fb3c.png)


If the user clicks "Yes", the index points will be deleted and the stage damage will be deleted.

This will be hard to test because we currently don't allow the user to create computed stage damages. I had to turn that off to test this, but am turning it back on.
